crates/common/src/lib.rs

```diff
@@ -294,22 +294,23 @@ impl TransactionVersion {
         u128::from_be_bytes(lower.try_into().expect("slice should be the right length"))
     }
 
+    pub const fn with_query_version(mut self) -> Self {
+        self.0 .0[15] |= 0b0000_0001;
+        self
```

Comment:
Formatter had a weird time I think
```suggestion
        self.0.0[15] |= 0b0000_0001;
        self
```

Reply:
I'm afraid it's pretty stubborn :sweat_smile: :
```
cargo fmt --all -- --check
Diff in /.../pathfinder/crates/common/src/lib.rs at line 295:
     }
 
     pub const fn with_query_version(mut self) -> Self {
-        self.0.0[15] |= 0b0000_0001;
+        self.0 .0[15] |= 0b0000_0001;
         self
     }
 
```